virttest/utils_zchannels.py

```diff
@@ -0,0 +1,168 @@
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
```

Comment:
I observe that avocado-vt is declared license as GPL-2.0 License. 
I wonder whether we need explicitly declare it again in newly created file.

Reply:
@chunfuwen Libvirt adds license to file header of each file, I took that and that we did include license declarations in other code files, too, as authorative. Please let me know if you want me to remove it.